packages/beacon-state-transition/src/cache/epochContext.ts

```diff
@@ -339,6 +365,37 @@ export class EpochContext {
       exitQueueChurn = 0;
     }
 
+    let previousTargetUnslashedBalanceIncrements = 0;
+    let currentTargetUnslashedBalanceIncrements = 0;
+    const forkSeq = config.getForkSeq(state.slot);
+    if (forkSeq > ForkSeq.phase0) {
+      const previousParticipation = (state as BeaconStateAltair).previousEpochParticipation.getAll();
+      for (const [i, flag] of previousParticipation.entries()) {
+        if ((flag & TIMELY_TARGET) === TIMELY_TARGET) {
+          const validator = state.validators.get(i);
+          if (isActiveValidator(validator, previousEpoch) && !validator.slashed) {
```

Comment:
isActiveValidator checks _should_ not be necessary, otherwise the attestation from which the status checks are derived would be invalid

Reply:
we have the same check in EpochProcess so I just follow. If I remove it, we can pass the random spec test, for example:

```
 60) minimal/bellatrix/rewards/random/pyspec_tests
       full_random_without_leak_0:
     Error: currentTargetUnslashedBalanceIncrements is wrong, expect 384, got 992,current epoch 7
```